### PR TITLE
fix default arg

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?
remove default list which can be dangerous usage

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by awaelchli** [comment]
is the `retry_jittered_backoff` not working well? I thought this was a feature.

**Comment by Borda** [comment]
> is the `retry_jittered_backoff` not working well? I thought this was a feature.

It was not used anywhere...

File: pytorch_lightning/trainer/distrib_parts.py [hunk 1]
```diff
@@ -748,26 +748,26 @@ def determine_root_gpu_device(gpus):
...
-            if i == num_retries - 1:
-                raise e
-            else:
-                continue
-        time.sleep(sleep)
-        sleep = min(cap, random.uniform(base, sleep * 3))
-
-
-def pick_single_gpu(exclude_gpus=[]):
+# def retry_jittered_backoff(f, num_retries=5):
```
Comment (williamFalcon) [inline pytorch_lightning/trainer/distrib_parts.py:751]: @Borda ?

Comment (Borda) [reply pytorch_lightning/trainer/distrib_parts.py:751]: is this used somewhere?

**Pull request merged**